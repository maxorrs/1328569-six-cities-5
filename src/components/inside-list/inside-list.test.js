import React from 'react';
import renderer from 'react-test-renderer';

import InsideList from './inside-list';

describe(`InsideList is rendered correctly`, () => {
  it(`InsideList is rendered correctly`, () => {
    const tree = renderer
      .create(
          <InsideList
            goods={[`A`, `B`, `C`]}/>
      )
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
