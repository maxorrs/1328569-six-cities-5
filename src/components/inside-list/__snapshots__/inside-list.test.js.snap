// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsideList is rendered correctly InsideList is rendered correctly 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      A
    </li>
    <li
      className="property__inside-item"
    >
      B
    </li>
    <li
      className="property__inside-item"
    >
      C
    </li>
  </ul>
</div>
`;
